### PR TITLE
feat: Implement dotnet6

### PR DESCRIPTION
**Pull request by lamonfly**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [X] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the new behavior?

Added support of dotnet6

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [X] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [ ] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## Checklist

Please check that your PR fulfills the following requirements:

- [X] Documentation has been added/updated.
- [ ] Automated Unit / Integration tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
- [ ] Your conventional commits are aligned with the **Impact on version** section.

### CODE REVIEW

File: src/DataLoader.Uno.WinUI/DataLoaderView.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.UI.Xaml;
```
Comment (jeanplevesque) [inline src/DataLoader.Uno.WinUI/DataLoaderView.cs:1]: Could we share code for `DataLoaderView `and `DataLoaderViewController` using global usings and base classes?

File: src/DataLoader.Uno/DataLoader.Uno.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
 		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
 		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;</TargetFrameworks>
```
Comment (jeanplevesque) [inline src/DataLoader.Uno/DataLoader.Uno.csproj:4]: You probably want to apply the same changes as @carlh98 did on other packages.

Comment (jeanplevesque) [reply src/DataLoader.Uno/DataLoader.Uno.csproj:4]: Like 
- target frameworks
- tags

File: DataLoader.sln [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿
```
Comment (jeanplevesque) [inline DataLoader.sln:1]: Make sure you also update the documentation (in readme).

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/DataLoader.Uno.WinUI/DataLoader.Uno.WinUI.csproj [hunk 4]
```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+	<PropertyGroup>
+		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
```
Comment (jeanplevesque) [inline src/DataLoader.Uno.WinUI/DataLoader.Uno.WinUI.csproj:4]: I think those are incorrect. Check with @carlh98

File: src/DataLoader.Uno.WinUI/DataLoader.Uno.WinUI.csproj [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>Chinook.DataLoader</RootNamespace>
+		<Authors>nventive</Authors>
+		<Company>nventive</Company>
+		<AssemblyName>Chinook.DataLoader.Uno.WinUI</AssemblyName>
+		<PackageId>Chinook.DataLoader.Uno.WinUI</PackageId>
+		<Description>Recipes for loading asynchronous data in a MVVM context.</Description>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<PackageTags>dataloader;async;mvvm;xamarin;ios;android;uwp;chinook</PackageTags>
```
Comment (jeanplevesque) [inline src/DataLoader.Uno.WinUI/DataLoader.Uno.WinUI.csproj:16]: please add **maui** and **winui** to this list of tags in **all csproj files** (except tests 😉)

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

**Review by jeanplevesque — APPROVED** [review]

**Review by carlh98 — APPROVED** [review]

**Pull request merged**